src/commands/git_secret_add.sh

```diff
@@ -74,6 +74,9 @@ function add {
       # see https://github.com/sobolevn/git-secret/issues/18 for more.
       _message "$message"
       _message "auto adding them to .gitignore"
+
+      # we need changes below for #789
+      # because this adds multiple filenames on one line
       for item in "${not_ignored[@]}"; do
```

Comment:
I guess that array expansion here does not work correctly for some reason 🤔 

Because `_add_ignored_file` looks correct to me: https://github.com/sobolevn/git-secret/blob/92d47c218de4e80da9f5a2681d23fe019a645520/src/_utils/_git_secret_tools.sh#L329